R/utils-get_code_dependency.R

```diff
@@ -33,6 +33,13 @@ get_code_dependency <- function(code, names, check_names = TRUE) {
     return(code)
   }
 
+  # If code is bound in curly brackets, remove them.
+  tcode <- trimws(code)
+  if (grepl("^\\{", tcode) && grepl("\\}$", tcode)) {
+    code <- gsub("^\\{", "", gsub("\\}$", "", tcode))
+  }
```

Comment:
```suggestion
  if (grepl("^\\{.*\\}$", tcode)) {
    code <- sub("^\\{(.*)\\}$", "\\1", tcode)
  }
```
feel free to ignore this.